scripts/ci/jsonSchemas/tutorial.json

```diff
@@ -40,6 +40,10 @@
       "type": "string",
       "description": "The image of the tutorial"
     },
+    "settings_rollback_needed": {
+      "type": "boolean",
+      "description": "Do we need to revert all the settings changes at the end of the tutorial?"
+    },
```

Comment:
I think unless we are going to alert users that the tutorials will change their settings or not, we should just make this default `true` and not even provide it as a property on the manifest.

Reply:
we are going to alert the user. This is a task for @MattLichtenstein